docs-src/references/configuration.md

```diff
@@ -210,7 +210,8 @@ Below is an example minimal specification for a password-secured Cluster using C
 ```yaml
 persistence:
   defaultStore: default
-  visibilityStore: visibility
+  visibilityStore: cass-visibility
+  secondaryVisibilityStore: es-visibility
```

Comment:
Why add secondary visibility config here?

Reply:
Because this is an example of persistence store config that can be used as a reference for the keys that can be set here. I'll add comments here to explain this better.